### PR TITLE
Consistent formatting for eval

### PR DESCRIPTION
**Pull request by phanimarupaka**
This PR is to introduce consistent formatting for eval command. If any function is invoked by eval, the resources are consistently formatted at the end.

https://github.com/GoogleContainerTools/kpt/issues/1973

### CODE REVIEW

**Review by Shell32-Natsu — APPROVED** [review]
lgtm, but want to wait for approval from Frank or Sunil

**Review by droot — APPROVED** [review]
Making it consistent with `render` seems ok to me.

**Pull request merged**
